pkg/stub/handler.go

```diff
@@ -155,11 +215,22 @@ func (h *Handler) StoreCurrentValidConfig(condition v1alpha1.SamplesResourceCond
 			}
 		}
 	}
+	cm.Data[regkey] = srcfg.Spec.SamplesRegistry
+	var value string
+	for _, val := range srcfg.Spec.SkippedImagestreams {
+		value = val + " "
+	}
+	cm.Data[skippedstreamskey] = value
+	value = ""
+	for _, val := range srcfg.Spec.SkippedTemplates {
+		value = val + " "
+	}
+	cm.Data[skippedtempskey] = value
```

Comment:
The comments in this function say we only update the ConfigMap once...   but it seems like you've introduced logic here and above that expects the ConfigMap to store the last applied configuration (skipped content, registry name).

Storing the last applied content makes sense, but then this other logic that expects to only create the configmap exactly once and not update it does not make sense.

Reply:
yep good catch ... need to fix comment, and move the setting of the install type and arch such that we only update that the first time, but allow for updates to the other fields multiple times.

part of next push